doc/docs/overview.md

```diff
@@ -8,20 +8,26 @@ The available options and information change after you **Select Device**.
 
 ## Select Device
 
-Opens the drop-down with the list of devices connected to the computer. You can choose a device from the list of connected devices to perform further actions on the device such as configuring.
+Once you connect a device to the system, it becomes visible and available when you click on the **Select device** drop-down list. You can choose a device from the list of connected devices to perform further actions on the device.
 
 ## Actions
 
-When you select a device, the serial port drop-down menu appears, where you can select the port for testing.
+When you select a device:
+
+* The **Serial port** and the **Baud rate** drop-down menus appear, where you can select the port for testing and the baud rate to use, respectively.
```

Comment:
serial port appears, yes, but baud rate selector is available without selecting a device